product_variant_tree_qty_available/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,3 @@
+* Ooops <https://ooops404.com>
```

Comment:
Please write here contributors names instead companies. If you want to make it appears, set them as titles with list of contributors under.

Like this : https://raw.githubusercontent.com/OCA/stock-logistics-warehouse/16.0/stock_inventory_count_to_zero/readme/CONTRIBUTORS.rst

Reply:
Hi @rousseldenis , in this case ooops is the client, Cetmix the developing company and Dessan Hemrayev is Cetmix's dev - so I think your suggestion can be applied to https://github.com/OCA/product-variant/pull/318/files#diff-41f15cc551357188322cc90b43cc6737430a4892f590d3995271812c934072f7R3 

Anyways there are lots of cases where only company is specified in contributors, and I can't seem to find specific guidelines on this either in https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id6 or https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/CONTRIBUTORS.md , so I think it would be better to agree on a common structure and then apply it.

Reply:
@rousseldenis ok now?

Reply:
@rousseldenis addressed, finally :)